src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -451,7 +451,7 @@ const formConfig = {
     formSignature: {
       title: 'Form signature',
       pages: {
-        supportingFilesReview: {
+        formSignature: {
```

Comment:
This page key was a duplicate (caused by copy paste in a previous PR). Just setting it to be a proper descriptive, unique name.